Hearthstone Deck Tracker/Core.cs

```diff
@@ -169,6 +173,38 @@ public static async void Initialize()
 			);
 		}
 
+		private static void CheckForHearthstoneUpdate()
+		{
+			try
+			{
+				var hearthDbVersion = HearthDb.Info.HearthDbVersion.ToString();
+				if(hearthDbVersion != Config.Instance.HearthdbVersion)
+				{
+					AssetDownloaders.cardImageDownloader.ClearStorage();
+					Config.Instance.HearthdbVersion = hearthDbVersion;
+					Config.Save();
+				}
+			}
+			catch(Exception ex)
+			{
+				Log.Error($"Could not check for updated HearthDB version: {ex}");
+			}
+			try
+			{
+				var hearthDbVersion = HearthDb.Info.HearthDbVersion.ToString();
+				if(RemoteConfig.Instance?.Data?.UpdateInfo?.Version != Config.Instance.RemoteHearthstoneVersion)
```

Comment:
Remote will not be loaded yet at this point. `RemoteConfig.Instance.Load` is called near the end of `Core.Initialize`. What you can do is move this second try-catch into it's own method and hook into the `RemoteConfig.Instance.Loaded` event *before* the `RemoteConfig.Instance.Load` call with this method. (`RemoteConfig.Instance.Loaded += YourNewCheck`)